pkg/appstore/app.go

```diff
@@ -26,3 +31,21 @@ func (a App) MarshalZerologObject(event *zerolog.Event) {
 		Str("version", a.Version).
 		Float64("price", a.Price)
 }
+
+func (a App) GetIPAName() string {
+	return fmt.Sprintf("%s+%s+%d+%s.ipa",
+		a.cleanName(a.BundleID),
+		a.cleanName(a.Name),
+		a.ID,
+		a.cleanName(a.Version))
```

Comment:
I would prefer to not include the app display name to not bother with altering it and replacing special characters. I don't see the value in doing so.

Let's go with the convention of `<bundle-id>_<app-id>_<version>.ipa` and just drop the random token. The regex cleanup will not be needed in this case I believe.

Reply:
The reason I added the display name is that some apps have completely unrelated bundle ids to the app name. This makes identifying such apps a real pain, especially if you have many in a single folder. For example, `Slack` is `com.tinyspeck.chatlyio`. Yes, the filename cleaning is not ideal, but IMO it adds a lot of value.

Reply:
Sure, the bundle ID might not resemble the app display name - but in that case, the user could just specify a custom file name using the `--output` flag. I see this as the responsibility of the user and not the tool. 🙂